templates/slim/section.html.slim

```diff
@@ -1,6 +1,9 @@
 / hide slides on %conceal, %notitle and named "!"
 - titleless = (title = self.title) == '!'
 - hide_title = (titleless || (option? :notitle) || (option? :conceal))
+
+/ slide options processing
+- data_background = (attr? 'background') ? (image_uri(attr 'background')) : nil
```

Comment:
Additionally (as I'm testing my old'ish slide decks with this PR), I was able to setup slides as follows

```asciidoc
[data-background="#eee", data-background-image=images/done-part1.gif, data-background-size=cover]
== No more Word design docs!
```

which resulted with a `section` having a class `has-light-background`. As per reveal.js release notes (version 3.0.0)

> The .reveal element is given a "has-light-background" or "has-dark-background" class when a per-slide background color is detected

Do you have any idea how it's possible now. I'm unable to set up a `class` attribute on `section` element, but having a `<section class="has-light-background">` will do the trick